app/src/main/java/com/team41/boromi/book/OwnedFragment.java

```diff
@@ -35,6 +38,7 @@
   private PagerAdapter pagerAdapter;
   private BookActivity bookActivity;
   private String parent = "Owned";
+  private String TAG = "OwenedFrag";
```

Comment:
OwnedFrag. Not really a big deal though...

Reply:
resolved